docker/test/stress/run.sh

```diff
@@ -131,13 +131,7 @@ function stop()
     # Preserve the pid, since the server can hung after the PID will be deleted.
     pid="$(cat /var/run/clickhouse-server/clickhouse-server.pid)"
 
-    # --max-tries is supported only since 22.12
-    if dpkg --compare-versions "$(clickhouse local -q 'select version()')" ge "22.12"; then
-        # Increase default waiting timeout for sanitizers and debug builds
-        clickhouse stop --max-tries 180 --do-not-kill && return
```

Comment:
Maybe we should keep increased timeout for BC check (because old versions do not have this improvement)

Reply:
Btw. I foresee that bc check will need more time to start. It will delete old parts from previous run at start now.

Reply:
The problem is that old version in BC check does not support `--max-tries` argument:
```

+ clickhouse stop --max-tries 180 --do-not-kill
std::exception. Code: 1001, type: boost::wrapexcept<boost::program_options::unknown_option>, e.what() = unrecognised option '--max-tries' (version 22.11.2.30 (official build))
```
https://s3.amazonaws.com/clickhouse-test-reports/44157/d54f739836fa998f0e32458bcaf3dad545e25526/stress_test__msan_.html

Reply:
And seems like it triggers gdb in BC check all the time and sometimes gdb hangs and it leads to "stress test without check_status.tsv "